Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -3105,10 +3105,15 @@ void StyleComputer::unload_fonts_from_sheet(CSSStyleSheet& sheet)
     }
 }
 
-static NonnullRefPtr<CSSStyleValue const> custom_property_initial_value(FlyString const& name)
+static NonnullRefPtr<CSSStyleValue const> custom_property_initial_value(DOM::Document& document, FlyString const& name)
```

Comment:
This seems like it should be a Document method as that's needed. I just put it here because I didn't know any better.